src/GoTrueApi.ts

```diff
@@ -336,7 +337,7 @@ export default class GoTrueApi {
   }
 
   /**
-   * Send User supplied Mobile OTP to be verified
+   * @deprecated Send User supplied Mobile OTP to be verified
```

Comment:
marked `verifyMobileOTP` as deprecated so we can potentially remove this in the future